doc/manual/de/config/customizing.rst

```diff
@@ -0,0 +1,273 @@
+.. _customizing:
+
+Anpassungen vornehmen
+=====================
+
+Die CometVisu kann auf viele verschiedene Weisen an die eigenen Bedürfnisse angepasst werden. Im Folgenden
+werden die verschiedenen Möglichkeiten - sortiert nach Komplexitätsgrad - im Detail beschrieben.
+
+===   =============================    ================    ===================================================
+\     Anpassung                        Veränderung          Beschreibung
+===   =============================    ================    ===================================================
+1.    Eigene CSS-Regeln                Optisch             Überschreiben der vorhandenen CSS-Regeln eines Designs durch das Laden von zusätzlichen CSS-Dateien
+2.    Eigene Icons einbinden           Optisch             Neben den mitgelieferten Icons können eigenen Icons über die Konfigurationsdatei hinzugefügt werden
+3.    Widgets durch ``class``          Optisch             Viele Widgets können individualisiert werden durch Hinzufügen des ``class`` Attributs und dazu passenden CSS-Regeln
+4.    Eigenes Design schreiben         Optisch             Sollten zusätzliche CSS-Regeln nicht ausreichen, kann ein neuen Design erstellt werden
+5.    Eigene Widgets schreiben         Inhaltlich          Hinzufügen neuer Widgets durch Plugins.
+6.    Eigene Struktur schreiben        Inhaltlich          Die Umwandlung der Konfigurationsdateien in HTML-Code, kann durch erstellen einer neuen Struktur komplett verändert werden
+===   =============================    ================    ===================================================
+
+Vorhandenes Design verändern
+----------------------------
+
+Die vorhandenen Designs können durch eigene CSS-Regeln angepasst werden. Im Meta-Bereich der Konfigurationsdatei
+können dazu Pfade zu CSS-Dateien angegben werden, die zusätzlich zu den CSS-Dateien des CometVisu-Designs geladen werden.
+
+.. code-block:: xml
+    :caption: Auszug einer Konfigurationsdatei, die 2 zusätzliche CSS-Dateien lädt:
+
+    <pages... design="metal">
+        <meta>
+            <files>
+                <file type="css">resource/my-custom-style.css</file>
```

Comment:
All those files should be located in `config/media/` as that's where the user is allowed to place any files and the manager supports that